apps/ui/components/application/ApplicationCard.tsx

```diff
@@ -27,21 +26,12 @@ const StyledButton = styled(Button)`
   }
 `;
 
-function getApplicant(application: ApplicationNameFragment, t: TFunction): string {
-  if (application.applicantType === ReserveeType.Individual) {
-    return t("applicationCard:person");
+function formatApplicantType(t: TFunction, application: Pick<ApplicationNameFragment, "applicantType">): string {
+  const { applicantType } = application;
+  if (!applicantType) {
+    return "";
   }
-  if (application.organisationName) {
-    return t("applicationCard:organisation", {
-      type: t(`applicationCard:applicantType.${application.applicantType?.toLocaleLowerCase()}`),
-      name: application.organisationName || t("applicationCard:noName"),
-    });
-  }
-  if (application.contactPersonFirstName) {
-    return t("applicationCard:person");
-  }
-
-  return "";
+  return t(`reservationApplication:reserveeTypes.labels.${applicantType}`);
```

Comment:
I guess on this card, if the type is other than `INDIVIDUAL`, the organisationName still needs to be appended after the translation; the separator could be a colon now, though; thus e.g. `On behalf of the company: Initrode` vs. `As a private person`.
This in case the same user creates applications on behalf of different organisations for the same round, so it's then possible to navigate to the correct one. (While there should not be more than 1 personal application so there's no point in distinguishing them by the contact person.)